### ISSUE TITLE
Possible regression in 2.0.0

### ISSUE THREAD

**Issue by simonrepp** [jGleitz/markdown-it-prism#17]
Hi!

First of all, thanks alot for creating the plugin!

I have possibly stumbled upon a regression in the new major release, basically after I update from `1.1.2` to `2.0.0` (and no other changes) my build process crashes with:
```
/home/simon/eno/eno-lang.org/node_modules/markdown-it/lib/index.js:496
  plugin.apply(plugin, args);
         ^

TypeError: plugin.apply is not a function
    at MarkdownIt.use (/home/simon/eno/eno-lang.org/node_modules/markdown-it/lib/index.js:496:10)
    at Object.<anonymous> (/home/simon/eno/eno-lang.org/lib/loaders.js:4:12)
    at Module._compile (internal/modules/cjs/loader.js:722:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:733:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:658:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/simon/eno/eno-lang.org/src/generators/demo.js:5:22)
```

This is the affected (short) file, problem happens on the call to `use`:
```js
const markdownIt = require('markdown-it')({ html: true });
const markdownItPrism = require('markdown-it-prism');

markdownIt.use(markdownItPrism);

exports.markdown = ({ value }) => markdownIt.render(value);
```

My version of `markdown-it` in use is `8.4.2`, `node` is at `v11.3.0`.

I've set up a branch in my repository for you where you can reproduce the error, switching back and forth between `master` and that branch (and updating the packages of course) demonstrates the regression on my machine:
https://github.com/eno-lang/eno-lang.org/tree/markdown-it-prism-reproduce

If you need more information, let me know.
Thanks!

**Comment by jGleitz** [comment]
Hi! 
This was definitely a regression. I introduced it by accidentally removing [babel-plugin-add-module-exports](https://www.npmjs.com/package/babel-plugin-add-module-exports) from the babel configuration.

I have published version 2.0.1 with a fix and deprecated version 2.0.0.

Thanks a lot for reporting this error! Please excuse the inconvenience it has caused.

**Comment by simonrepp** [comment]
That's great news :) - will update this week sometime!

Thanks for the quick fix!
